composer/loggers/remote_uploader_downloader.py

```diff
@@ -41,7 +41,8 @@ def _build_remote_backend(remote_backend_name: str, backend_kwargs: Dict[str, An
         's3': S3ObjectStore,
         'oci': OCIObjectStore,
         'sftp': SFTPObjectStore,
-        'libcloud': LibcloudObjectStore
+        'libcloud': LibcloudObjectStore,
+        'gcs': GCSObjectStore,
```

Comment:
I think I'm missing something. How does this work? won't the `remote_backend_name` be `gs` which does not exist in this dictionary?

Reply:
Yes you are right. it needs to be the scheme name. I guess this was the original reason I use gs across the files.....